packages/storybook/CallComposite/__snapshots__/BasicExample.stories.storyshot

```diff
@@ -0,0 +1,44 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Why did so many stories get added?

Reply:
Oh I think you'll need to remove these and do a `rush update` before updating the snapshots